hwdb/60-evdev.hwdb

```diff
@@ -114,8 +114,8 @@ evdev:name:ETPS/2 Elantech Touchpad:dmi:bvn*:bvr*:bd*:svnASUSTeKComputerInc.:pnK
  EVDEV_ABS_35=::18
  EVDEV_ABS_36=::16
 
-# Asus X550CC and S550CB
-evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pn?550C?:*
+# Asus X550CC, S550CB and N550JV
+evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pn?550??:*
```

Comment:
I think it'd be safer to add a separate line. Assuming that all ASUSes with 550 in the name are the same might break in the future.

@whot?

Reply:
Ok, I can make a separate rule. Given that I am at it, do you want me to break also X550CC and S550CB in two separate rules? I assume the same principle applies.

Reply:
That'd be great.